packages/app-security/src/admin/plugins/secureRouteError.tsx

```diff
@@ -9,10 +18,13 @@ const plugin: SecureRouteErrorPlugin = {
     name: "secure-route-error",
     render() {
         return (
-            <>
+            <div className={notAuthCss}>    
                 <Helmet title={"Not authorized"} />
-                <span>You are not authorized to access this route.</span>
-            </>
+                <img src={authErrorImg} alt="Not Authorized" width="40%"/>
```

Comment:
Please remove the inline style here. We avoid using inline style as much possible. You can use `CSS` class or even [styled component](https://emotion.sh/docs/styled)